client/cypress/integration/pages/userprofile.spec.js

```diff
@@ -1,10 +1,9 @@
 describe("Check user profile", () => {
-  before(() => {
-    cy.seedDb();
-  });
-
   beforeEach(() => {
     cy.visit("/profil");
+    // TODO: Is the seeding mandatory? It costs time…
+    // How to make the tests self containing otherwise?
+    cy.seedDb();
```

Comment:
Wenn du in :119 nicht den "normal" User aus dem Command nimmst, sondern einen anderen User einloggst und dessen Passwort änderst, reicht ein seedDb in before().

Reply:
werde ich ändern